pyo3-ffi/src/cpython/frameobject.rs

```diff
@@ -89,7 +89,8 @@ extern "C" {
     pub fn PyFrame_FastToLocals(f: *mut PyFrameObject);
 
     // skipped _PyFrame_DebugMallocStats
-    // skipped PyFrame_GetBack
+    #[cfg(all(Py_3_9, not(PyPy)))]
+    pub fn PyFrame_GetBack(f: *mut PyFrameObject) -> *mut PyFrameObject;
```

Comment:
It looks like this function was moved to `pyframe.h` in upstream CPython. We should log a follow up to sync this sometime.